helpers/pdfUtils.py

```diff
@@ -0,0 +1,20 @@
+import logging
+import tempfile
+import subprocess
+from django.conf import settings
+from subprocess import CalledProcessError
+
+logger = logging.getLogger(__name__)
+
+SCRIPT_PATH = settings.BASE_DIR + "/bin/generate-pdf.js" 
+
+def get_pdf_for_url(url):
+  try:
+    tmp = tempfile.NamedTemporaryFile(delete=True)
+    child = subprocess.Popen(["node", SCRIPT_PATH, "-w", tmp.name, "-u %s" % (url)], stdout=subprocess.PIPE)
```

Comment:
Docs suggest `subprocess.call` is simplest for common uses, are there specific reasons we need to use `Popen`? https://docs.python.org/2/library/subprocess.html

Reply:
Its a holdover from a previous version of this function where the PDF contents were read from STDOUT. Will switch to subprocess.call